invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.variables

```diff
@@ -7,3 +7,4 @@
 // Tablets/phones were left as-is because they have less screen estate
 @selectionComputerMaxMenuHeight : 575px;
 @selectionWidescreenMaxMenuHeight : 575px;
+@selectionIconPadding: calc(@selectionIconHitbox / unit(@dropdownIconSize));
```

Comment:
This variable is defined in [semantic ui less](https://github.com/Semantic-Org/Semantic-UI-LESS/blob/master/themes/default/modules/dropdown.variables#L163) but it's not properly compiled. I see that we have done some major upgrades lately ([here the commit](https://github.com/inveniosoftware/invenio-assets/commit/656f0fe4943b748bcec54e2f6c4bce4b1b3bb22a#diff-e0c1de0eb18d1877eced2250b1c0583e65790027d37eed938d8e8f80aa8dea0aL14-L66)). I've quickly checked the changelog for some packages that might be involving in the loading of the css such as `mini-css-extract-plugin`, `less-loader` and `css-loader` but couldn't find the reason why this is failing now. Any advice on what else to check is welcomed!

Reply:
Dsicussed IRL, we fix the inconsistencies that we find due to the upgrade